### PR TITLE
Preparation for HACS and basic documentation

### PR DESCRIPTION
**Pull request by agdamsbo**
Preparing for HACS for easier use, updating and testing, and updated basic documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,66 @@
...
+## Available actions
+
+### Play Latest
+Play the latest video or clip from a specific program. There exists two options to exclude or include videos matching specific categories.
+```yaml
+- service: drtv_play.play_latest
+  entity_id: media_player.living_room_tv
+  data:
+    program_name: gurli
+    subtitles: false # Optional, default is false
```
Comment (TermeHansen) [inline README.md:14]: subtitles is on play_channel only (subtitles are part of the hls video stream),

File: README.md [hunk 2]
```diff
@@ -1,2 +1,66 @@
-# home-assistant-drtv-play
-Play DRTV videos and channels via home assistant
+# Home Assistant DRTV Play
+
+Play DRTV videos and channels via Home Assistant
+
+## Available actions
+
+### Play Latest
+Play the latest video or clip from a specific program. There exists two options to exclude or include videos matching specific categories.
```
Comment (TermeHansen) [inline README.md:8]: there is no excluding or categories here. But maybe a line on either general search or path (starting with ´/' or id (being an integer)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,66 @@
...
+    subtitles: false # Optional, default is false
+```
+
+### Play Channel
+Play one of the svt channels.
+```yaml
+- service: drtv_play.play_channel
+  entity_id: media_player.living_room_tv
+  data:
+    channel: dr2 # Available channels: dr1, dr2, drtv, drtv ekstra, dr ramasjang
```
Comment (TermeHansen) [inline README.md:23]: missing subtitles here

**Comment by agdamsbo** [comment]
I went through it all again. It shouldn't say svt anymore..

File: README.md [hunk 4]
```diff
@@ -1,2 +1,63 @@
...
+[![unit testing in python](https://github.com/TermeHansen/home-assistant-drtv-play/actions/workflows/python-package-conda.yml/badge.svg)](https://github.com/TermeHansen/home-assistant-drtv-play/actions/workflows/python-package-conda.yml)
+
+# Home Assistant DRTV Play
+
+Play DRTV videos and channels via Home Assistant
+
+## Available actions
+
+### Play Latest
+Play the latest video or clip from a specific program. You can search by entering a string (eg. "gurli"), or by writing the path (eg. /alene-i-vildmarken) or id (integer, eg. 69758) from the url.
```
Comment (TermeHansen) [inline README.md]:
```suggestion
Play the latest episode from a specific show. You can search by entering any string (eg. "gurli"), or by writing the ending path from https://www.dr.dk/drtv/ (eg. "/serie/alene-i-vildmarken_69758", it should start with '/') or id (integer, eg. 69758) from the url. 
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,63 @@
...
+Play the latest video or clip from a specific program. You can search by entering a string (eg. "gurli"), or by writing the path (eg. /alene-i-vildmarken) or id (integer, eg. 69758) from the url. 
+```yaml
+- service: drtv_play.play_latest
+  entity_id: media_player.living_room_tv
+  data:
+    program_name: gurli
+```
+
+### Play Channel
+Play one of the DRTV channels. Available channels are DR 1, DR 2, DRTV, DRTV Ekstra and DR Ramasjang.
```
Comment (TermeHansen) [inline README.md]:
```suggestion
Play one of the DRTV channels. Available channels are DR1, DR2, DRTV, DRTV Ekstra and DR Ramasjang.
```

**Review by TermeHansen — APPROVED** [review]

**Pull request merged**